### PR TITLE
Create a static storybook builder

### PR DESCRIPTION
**Pull request by thani-sh**

### CODE REVIEW

**Comment by mnmtanish** [comment]
**NOT READY**

File: src/server/index.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ app.get('/', function (req, res) {
   res.send(getIndexHtml());
 });
 
-app.get('/iframe', function (req, res) {
+app.get('/iframe.html', function (req, res) {
```
Comment (Markchri34) [inline src/server/index.js]: !https://travis-ci.org/kadirahq/react-storybook.svg?branch=static-build!:https://travis-ci.org/kadirahq/react-storybook

**Comment by mnmtanish** [comment]
@arunoda I think this one's ready

File: src/server/webpack.config.prod.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import path from 'path';
+import webpack from 'webpack';
+
+const config = {
+  devtool: 'cheap-module-source-map',
```
Comment (arunoda) [inline src/server/webpack.config.prod.js]: This needs `#`. See: https://github.com/kadirahq/react-storybook/pull/85

File: src/server/webpack.config.prod.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    preview: [
+      path.resolve(__dirname, '../client/init_preview'),
+    ],
+  },
+  output: {
+    filename: '[name].bundle.js',
+    publicPath: '/static/',
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
```
Comment (arunoda) [inline src/server/webpack.config.prod.js]: Add a minify plugin.

File: dist/builder/build.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
```
Comment (arunoda) [inline dist/builder/build.js]: We don't need this file right?
May be try to re-add `dist` directory.

**Comment by arunoda** [comment]
@mnmtanish Everything works pretty well. But when we used a custom config file, things get bit weird.
See:

screen shot 2016-04-11 at 7 04 36 pm

Since these modules load dynamically. We may need to do something about this.

**Comment by arunoda** [comment]
Interestingly, this works pretty well with the [simple-demo](https://github.com/kadira-samples/react-storybook-simple-demo) but not with the [todo-mvc-app](https://github.com/kadira-samples/react-storybook-demo).

**Pull request merged**